tools/cli/internal/openapi/oasdiff.go

```diff
@@ -417,6 +483,27 @@ func (o OasDiff) areSchemaIdentical(name string) bool {
 	return !ok
 }
 
+// arePathsIdenticalWithExcludeExtensions checks if the paths are identical with the extensions excluded
```

Comment:
could you provide an example of the extension in the comment?

Reply:
this runs oasdiff with an "ignore extensions" config, I've updated the comment